src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -362,3 +362,10 @@
     validator=validators.validate_autostop_error_rate_time_window,
     help="Time window during which the error percentage should be evaluated in seconds.",
 )
+
+regionwise_engines = CLIArgumentType(
+    options_list=["--regionwise-engines"],
+    validator=validators.validate_regionwise_engines,
+    nargs="+",
+    help="Regionwise engine count in the format of region1=engineCount1 region2=engineCount2 ... The region name should be of format accepted by ARM, and should be a region supported by Azure Load Testing. Multi-region load tests can only be run against public endpoints.",
```

Comment:
@saloniagrawal1997 please help review

Reply:
Specify the engine count for each region in the format: region1=engineCount1 region2=engineCount2 .... Use region names in the format accepted by Azure Resource Manager (ARM). Ensure the regions are supported by Azure Load Testing. Multi-region load tests can only target public endpoints